app/modules/submissions/models.py

```diff
@@ -280,15 +282,19 @@ def import_tus_files(self, transaction_id=None):
                 )
                 os.rename(os.path.join(root, name), os.path.join(submission_path, name))
 
+        assets_added = []
         if num_files > 0:
             log.info('Tus collect for %d files moved' % (num_files))
             self.git_commit('Tus collect commit for %d files.' % (num_files,))
             self.git_push()
+            for asset in self.assets:
+                if asset.path in paths_added:
+                    assets_added.append(asset)
 
+        # i _kind of_ think we can rmdir() for _any_ case.  have to think about this.   -jon
```

Comment:
I wasn't certain so I only did it for transactions

Reply:
i now used `shutil.rmtree()` to kill the dir _even if files are in it_.  this can happen in the case where optional `path=` is passed in -- in which case only a subset of the files in the dir are made into assets.  the others left behind go to bit heaven.